images/demo/user-home/02 - Web3 Data (Ethereum trading example)/01 - Working with Web3 data.ipynb

```diff
@@ -422,7 +425,7 @@
     "          &page=1\\\n",
     "          &offset=1000\\\n",
     "          &startblock=0\\\n",
-    "          &endblock=99999999\n",
+    "          &endblock=99999999\"\n",
```

Comment:
just in case, is this an expected change?

Reply:
yep, there was an unclosed quote